DESCRIPTION

```diff
@@ -21,7 +21,6 @@ Imports:
     magrittr,
     purrr,
     rlang,
-    stringdist,
```

Comment:
This whole package is basically a bit of glue code around stringdist compare https://github.com/2DegreesInvesting/tilt.company.match/blob/05351998b78b37eca3108253f52691f344c25742/README.Rmd#L151 so it cannot be simply removed.
Can you please elaborate on why you want to remove it?

Reply:
You're right. I added again 'stringdist' to DESCRIPTION but note it can't go under `Imports:` but under `Suggests:`.

R CMD checks that packages listed in `DESCRIPTION > Imports:` are used somewhere in the R/ directory. If you use a package in tests, vignettes, or README but not in the R/ directory, then you may list it in `DESCRIPTION > Suggests:`.

Reply:
Just to go sure, if the package is under suggests will it still install automatically as a dep for the user, same as if it is in imports? Since if that is not the case I fear this will be one round of tech support needed per partner so I would recommend to rather live with the Note. (Or ideally just put the respective bit from the readme in a tested exported function of course :)

Reply:
From what you say I think the best place is DESCRIPTION > Depends:. 

That's a light weight way of creating a meta package (idea from Jim Hester former tidyverse team member). That strongly and succinct advertises the intent, and keeps R CMD check happy.

Reply:
I confirm it works as I expect. But while I was doing this I noticed we still need to import at least one function from stringdist to keep R CMD check happy. And with that, then we can also move stringdist to `DESCRIPTION > Imports:` and pass R CMD check cleanly. So if you prefer it in `Imports:` I can move it back there. But then I would load it right after this package to helps users see all the packages they need with one glance. If we keep it in `Depends:` we don't need that because it loads automatically with this package.